src/Umbraco.Web.UI.Client/src/views/packager/delete.controller.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @ngdoc controller
+ * @name Umbraco.Editors.Packages.DeleteController
+ * @function
+ *
+ * @description
+ * The controller for deleting content
+ */
+function PackageDeleteController($scope, packageResource, treeService, navigationService) {
+
+    $scope.performDelete = function() {
+
+        //mark it for deletion (used in the UI)
+        $scope.currentNode.loading = true;
+        packageResource.deleteCreatedPackage($scope.currentNode.id).then(function () {
+            $scope.currentNode.loading = false;
+             
+            //get the root node before we remove it
+            var rootNode = treeService.getTreeRoot($scope.currentNode);
+
+            //TODO: Need to sync tree, etc...
+            treeService.removeNode($scope.currentNode);
```

Comment:
Is the TODO still outstanding @Shazwazza it would seem with the `removeNode` it should be enough?